lib/cocoapods/generator/copy_resources_script.rb

```diff
@@ -100,7 +100,7 @@ def script
 set -o pipefail
 
 if [ -z ${UNLOCALIZED_RESOURCES_FOLDER_PATH+x} ]; then
-    # If UNLOCALIZED_RESOURCES_FOLDER_PATH is not set, then there's no where for us to copy
+    # If UNLOCALIZED_RESOURCES_FOLDER_PATH is not set, then there's nowhere for us to copy
```

Comment:
need to bump integration specs for that :)

Reply:
https://github.com/CocoaPods/cocoapods-integration-specs/pull/137